README.md

```diff
@@ -1,133 +1,57 @@
 
 ```bash
-# Create the build directory and change into it
-mkdir -p build && cd build
-
-# Configure CMake from the command line passing configure-time options. 
-cmake .. -DCMAKE_BUILD_TYPE=Release -DSEATBELTS=OFF -DCUDA_ARCH=70 -DVISUALISATION=ON
-
-# Build the target(s)
-cmake --build . --target submodel_benchmark -j 8
+# Configure 
+cmake . -B build -DCMAKE_BUILD_TYPE=Release -DSEATBELTS=OFF -DBUILD_SWIG_PYTHON=OFF -DCUDA_ARCH=70
+cmake --build build -j`nproc` 
+make
 ```
 
-### Windows
-
-Under Windows, you must instruct CMake on which Visual Studio and architecture to build for, using the CMake `-A` and `-G` options.
-This can be done through the GUI or the CLI.
-
-To configure and build the `submodel_benchmark` binary on linux for benchmarking using Volta generation (SM 70) GPU(s), configure as a Release build with SEATBELTS disabled via :
-
-```cmd
-REM Create the build directory 
-mkdir build
-cd build
+### Execution and Data generation
 
-REM Configure CMake from the command line, specifying the -A and -G options. Alternatively use the GUI
-cmake .. -A x64 -G "Visual Studio 16 2019"-DSEATBELTS=OFF -DCUDA_ARCH=70 
+The `FLAMEGPU2_INC_DIR` environment variable must be set for run-time compilation (TRC) to succeed. This can be done at the same time as execution of the model. E.g. 
 
-REM You can then open Visual Studio manually from the .sln file, or via:
-cmake --open . 
-REM Alternatively, build from the command line specifying the build configuration
-cmake --build . --config Release --target submodel_benchmark --verbose
-```
-
-Alternatively to configure for visualisation purposes, enable the `VISUALISATION` CMake option:
-
-```cmd
-REM Create the build directory 
-mkdir build
+```bash
 cd build
-
-REM Configure CMake from the command line, specifying the -A and -G options. Alternatively use the GUI
-cmake .. -A x64 -G "Visual Studio 16 2019"-DSEATBELTS=OFF -DCUDA_ARCH=70 -DVISUALISATION=ON
-
-REM You can then open Visual Studio manually from the .sln file, or via:
-cmake --open . 
-REM Alternatively, build from the command line specifying the build configuration
-cmake --build . --config Release --target submodel_benchmark --verbose
+FLAMEGPU2_INC_DIR=./_deps/flamegpu2-src/include/ ./bin/Release/submodel-benchmark 
 ```
 
-## Running the Benchmark or Visualisation
+This will produce 6 `.csv` files in the `build` directory.
 
+## Plotting Results
 
-Once compiled, executing the generated binary file will run the performance benchmark, or run the visualisation of a single infinitely running simulation (at a reduced rate of simulation)
+### Dependencies
 
-I.e. from the `build` directory on Linux:
-
-```bash
-./bin/Release/submodel_benchmark 
-```
-
-For non-visualisation builds, this will generate a CSV file with performance metrics included which can later be post-processed to generate figures.
-
-
-## Plotting Benchmark Data
 
 Figures can be generated from data in CSV files via a python script.
```

Comment:
Move this above the dependencies subheading.

Reply:
done